src/main/java/cist4830/unomaha/tempo/controllers/ErrorHandlerController.java

```diff
@@ -0,0 +1,17 @@
+package cist4830.unomaha.tempo.controllers;
+
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ErrorHandlerController implements ErrorController {
+
+    @Override
+    @RequestMapping("/error")
+    @ResponseBody
+    public String getErrorPath() {
```

Comment:
I believe we discussed this at some point, but having two different options based on whether the user was administrative or not could lead to different errors, e.g. a stack trace if we are logged in to an administrator account (just as a concept).

Reply:
Yeah, that's a really great idea.  Just not sure how to get there just yet.  I commented this out for dev builds, but for prod it should exist (at least until we figure out how to toggle it)